### ISSUE TITLE
Resign option in multiplayer

### ISSUE THREAD

**Issue by GRhin** [Sh0wdown/UnofficialCrusaderPatch#519]
Hi,
Now that we have ability to add menu items, would it be possible to add an option the the menu in multiplayer for "Resign", currently its "end this game?" or "Quit", one requires everyone to agree, the other quits and doesnt show the stats page. However when you are playing 3+ teams, sometimes you want to quit, but still watch the rest of the game. 
My thinking is that when you click "resign" your lord is insta-killed.

**Comment by Krarilotus** [comment]
Killed by whom? You usually collect gold and ressources for a kill.  I think this option sounds good on paper but actually for many party games, people should just kill the lord. Thats not to difficult and you can also just send him out if you want to do that on purpose.
I think GG in the chat + end this game is a good option. In multiparty games, people should still kill the lord. There is nothing to preventing a 'dead' player from watching while his lord is still alive.

**Comment by GRhin** [comment]
Fair point, I'm also thinking of the lobby software im wanting to set up and how it knows what a "win" is, but that's another issue entirely.
However, the game does seem to notice who us "responsible " for killing somebody, as if I suicide my lord (to watch a match) it sometimes gives the gold to a nearby lord who shot some of my units or villagers before I died.... not sure how accurate itd be though, maybe it is just who hurt you last...

**Comment by J-T-de** [comment]
You can get killed by lions, so I guess killed by no enemy player would be possible?

**Comment by GRhin** [comment]
Yes, I have been a watcher of several games, so I deliberately die either by suicide or by lions placed on the map for that purpose, and what I have noticed is that if I die having never been attacked by another player, no one gets the reward, but if i am attacked in some way before i die, someone gets the reward. I have not looked into enough to know why it gives it to each player, but i suspect it is who last hurt me.
